refellips/structureSE.py

```diff
@@ -346,6 +346,86 @@ def complex(self, wavelength):
         return real + 1j * 0.0
 
 
+class Sellmeier(ScattererSE):
+    r"""
+    Dispersion curves for Sellmeier oscillators.
+
+    Parameters
+    ----------
+    Am: {float, Parameter, sequence}
```

Comment:
As implemented a sequence would fail. Gauss and Lorentz accept sequence of float/Parameter. Does CompleteEase have a series of oscillators? If it does then we should implement a series of oscillators. If not, then the docstring needs to be changed to `{float, Parameter}`.

Reply:
CompleteEase only offers a single oscillator. But there are other examples on the web where a sequence is used. Equations are not consistent so I just chose the CompleteEase version